tvtk/tests/test_vtk_parser.py

```diff
@@ -45,8 +45,13 @@ def test_parse(self):
         p = self.p
         # Simple case of a vtkObject.
         p.parse(vtk.vtkObject())
-        self.assertEqual(p.get_toggle_methods(),
-                         {'Debug': 0, 'GlobalWarningDisplay': 1})
+        if vtk_major_version < 9:
+            self.assertEqual(p.get_toggle_methods(),
+                             {'Debug': 0, 'GlobalWarningDisplay': 1})
+        else:
+            self.assertIn(p.get_toggle_methods(),
+                          [{'Debug': False, 'GlobalWarningDisplay': 1},
+                           {'Debug': False, 'GlobalWarningDisplay': 0}])
```

Comment:
Also no idea why GlobalWarningDisplay can take on two values for VTK9

Reply:
It is possible some other test sets the warning state to 0, so there is some issue.